.github/workflows/main.yml

```diff
@@ -91,7 +91,7 @@ permissions:
 jobs:
   build-graalvm-linux:
     name: /${{ matrix.env.PRIMARY }} ${{ matrix.env.GATE_TAGS }} JDK${{ matrix.env.JDK_VERSION }}
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Can we keep `ubuntu-20.04` for now?

Reply:
Yes of course, mistake on my side! 
Out of curiosity, any specific reason not to use 22.04?

Reply:
Yes: we want to test with the oldest glibc available (ubuntu-20.04 is the oldest ubuntu currently supported by GHA), so that we know things are still working there. The moment the glibc version gets bumped, we will know because the builds will fail and then will we move to 22.04. The same is true for any other dev and non-dev dependency.

Reply:
I see, thanks for the clarification :smile: